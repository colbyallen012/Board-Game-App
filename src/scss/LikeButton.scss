 .like-btn {
  margin: 6px;
  justify-content: flex-start;
}
