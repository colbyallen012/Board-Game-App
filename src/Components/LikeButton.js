import React, { Component } from 'react';
import '../scss/LikeButton.scss';

class LikeButton extends Component {
  constructor(props) {
    super(props);

  }

  render() {
    return(
      <div>
        <img src={this.props.unlike} className="like-btn"></img>
      </div>
    );
  }
}

export default LikeButton;
