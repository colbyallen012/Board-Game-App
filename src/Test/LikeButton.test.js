import React from 'react';
import ReactDOM from 'react-dom';
import LikeButton from '../Components/LikeButton';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

const handleClick= jest.fn()


describe('LikeButton', () => {
  let wrapper;
  
  beforeEach(() =>{
    wrapper = shallow(
      <LikeButton
      />
    )
  })


  it('should match the snapshot with all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it.skip('should toggle Favorite Status', () => {
    const instance = wrapper.instance();
    expect(wrapper.props('isFavorite')).toEqual(false);
    wrapper.find('.like-btn').simulate('click');
    expect(wrapper.props('isFavorite')).toEqual(true);
  });

})