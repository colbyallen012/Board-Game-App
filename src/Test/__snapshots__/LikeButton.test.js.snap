// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LikeButton should match the snapshot with all data passed in correctly 1`] = `
<div>
  <img
    className="like-btn"
    onClick={[Function]}
  />
</div>
`;
